### ISSUE TITLE
Air Source Heat Pump htg_coil

### ISSUE THREAD

**Issue by joseph-robertson** [NREL/OpenStudio-Beopt#24]
The blank Rated Supply Fan Power Per Volume Flow Rate below does not seem to be in the SDK. (BEopt writes Rated Evaporator Fan Power Per Volume Flow Rate.)

```python
Coil:Heating:DX:SingleSpeed,
  DX Heating Coil,                        !- Name
  HeatingSeasonSchedule,                  !- Availability Schedule Name
  3516.85284207,                          !- Gross Rated Heating Capacity {W}
  3.33368305593386,                       !- Gross Rated Heating COP {W/W}
  Autosize,                               !- Rated Air Flow Rate {m3/s}
  ,                                       !- Rated Supply Fan Power Per Volume Flow Rate {W/(m3/s)}
  Forced Air System Cooling Coil - Heating Coil Node, !- Air Inlet Node Name
  Forced Air System Heating Coil - Supplemental Coil Node, !- Air Outlet Node Name
  HP_Heat-Cap-fT,                         !- Heating Capacity Function of Temperature Curve Name
  HP_Heat-CAP-fFF,                        !- Heating Capacity Function of Flow Fraction Curve Name
  HP_Heat-EIR-fT,                         !- Energy Input Ratio Function of Temperature Curve Name
  HP_Heat-EIR-fFF,                        !- Energy Input Ratio Function of Flow Fraction Curve Name
  HP_Heat-PLF-fPLR,                       !- Part Load Fraction Correlation Curve Name
  DefrostEIR,                             !- Defrost Energy Input Ratio Function of Temperature Curve Name
  -17.7777777777777,                      !- Minimum Outdoor Dry-Bulb Temperature for Compressor Operation {C}
  ,                                       !- Outdoor Dry-Bulb Temperature to Turn On Compressor {C}
  4.44444444444446,                       !- Maximum Outdoor Dry-Bulb Temperature for Defrost Operation {C}
  0,                                      !- Crankcase Heater Capacity {W}
  10,                                     !- Maximum Outdoor Dry-Bulb Temperature for Crankcase Heater Operation {C}
  ReverseCycle,                           !- Defrost Strategy
  OnDemand,                               !- Defrost Control
  0.166667,                               !- Defrost Time Period Fraction
  2000;                                   !- Resistive Defrost Heater Capacity {W}
```

**Comment by kbenne** [comment]
@DavidGoldwasser it just didn't exist when we originally added the coil object. I'm adding it right now.

**Comment by kbenne** [comment]
@shorowit Looks like ```Rated Supply Fan Power Per Volume Flow Rate``` is now ```Rated Evaporator Fan Power Per Volume Flow Rate```

**Comment by kbenne** [comment]
And actually on closer review it looks like ```Rated Evaporator Fan Power Per Volume Flow Rate``` is already supported. Perhaps there is no action here. @shorowit and @joseph-robertson Please comment if you agree or not.

Here is what we are getting out of OS...

```

Coil:Cooling:DX:SingleSpeed,
  Coil Cooling DX Single Speed 1,         !- Name
  Always On Discrete,                     !- Availability Schedule Name
  Autosize,                               !- Gross Rated Total Cooling Capacity {W}
  Autosize,                               !- Gross Rated Sensible Heat Ratio
  3,                                      !- Gross Rated Cooling COP {W/W}
  Autosize,                               !- Rated Air Flow Rate {m3/s}
  773.3,                                  !- Rated Evaporator Fan Power Per Volume Flow Rate {W/(m3/s)}
  Node 8,                                 !- Air Inlet Node Name
  Node 9,                                 !- Air Outlet Node Name
  Curve Biquadratic 1,                    !- Total Cooling Capacity Function of Temperature Curve Name
  Curve Quadratic 1,                      !- Total Cooling Capacity Function of Flow Fraction Curve Name
  Curve Biquadratic 2,                    !- Energy Input Ratio Function of Temperature Curve Name
  Curve Quadratic 2,                      !- Energy Input Ratio Function of Flow Fraction Curve Name
  Curve Quadratic 3,                      !- Part Load Fraction Correlation Curve Name
  ,                                       !- Nominal Time for Condensate Removal to Begin {s}
  ,                                       !- Ratio of Initial Moisture Evaporation Rate and Steady State Latent Capacity {dimensionless}
  ,                                       !- Maximum Cycling Rate {cycles/hr}
  ,                                       !- Latent Capacity Time Constant {s}
  ,                                       !- Condenser Air Inlet Node Name
  AirCooled,                              !- Condenser Type
  0,                                      !- Evaporative Condenser Effectiveness {dimensionless}
  ,                                       !- Evaporative Condenser Air Flow Rate {m3/s}
  ,                                       !- Evaporative Condenser Pump Rated Power Consumption {W}
  0,                                      !- Crankcase Heater Capacity {W}
  0,                                      !- Maximum Outdoor Dry-Bulb Temperature for Crankcase Heater Operation {C}
  ,                                       !- Supply Water Storage Tank Name
  ,                                       !- Condensate Collection Water Storage Tank Name
  0,                                      !- Basin Heater Capacity {W/K}
  10;                                     !- Basin Heater Setpoint Temperature {C}

```

**Comment by kbenne** [comment]
doh! i think i have mashed potato brain. We are talking about heating coil here. ugh... sorry

**Comment by kbenne** [comment]
Done https://github.com/NREL/OpenStudio/commit/070a62c531522c85fac10a1be2a9f6069e7269d0

**Comment by joseph-robertson** [comment]
Confirmed that this is fixed in 1.11.5.
